src/Views/Results/Index.cshtml

```diff
@@ -16,7 +16,7 @@
   </div>
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-one-third">
-      <button data-target="#searchFilters" class="js-toggle">
+      <button data-target="#searchFilters" class="js-toggle" aria-expanded="false">
```

Comment:
Shouldn't the `aria-expanded` be on the target, then this should have `aria-controls`?

Reply:
No, `aria-expanded` should be on the trigger as per https://www.w3.org/WAI/GL/wiki/Using_aria-expanded_to_indicate_the_state_of_a_collapsible_element#Description. I can definitely add an `aria-controls` attribute though 👌